recipes/openfermionpyscf/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "openfermionpyscf" %}
+{% set version = "0.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/openfermionpyscf-{{ version }}.tar.gz
+  sha256: 60b7a8a03e4e2415b5e5b4f8aed02203add0fe90eafff5873580cf2d6d67729f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - openfermion >=0.5
+    - pyscf
+    - pytest
```

Comment:
Yes, this is really listed as required for some reason.

Reply:
I've added that comment so often when reviewing... :)

Yes but that's not yet the case for 0.5, see here: https://github.com/quantumlib/OpenFermion-PySCF/blob/v0.5/requirements.txt

Will remove that with the next update though!

Reply:
I see! You want `pip check` to pass, so it has to be there.